py-client test_snapshot failure

The following error has been observed a few times recently in a few GH jobs:

```
======================================================================
FAIL [0.091s]: test_snapshot (test_table.TableTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/project/tests/test_table.py", line 222, in test_snapshot
    self.assertGreaterEqual(test_table.size, result_table.size)
AssertionError: 0 not greater than or equal to 1
```

The underlying code looks suspicious:

```python
        test_table = self.session.time_table(period=10000000).update(formulas=["Col1 = i", "Col2 = i * 2"])
        result_table = test_table.snapshot()
        self.assertEqual(test_table.schema, result_table.schema)
        self.assertGreaterEqual(test_table.size, result_table.size)
```

Afaict, table.size is only ever set once, so the `assertGreaterEqual` check seems racy at best, and completely wrong at worst.

--- a/py/client/tests/test_table.py
+++ b/py/client/tests/test_table.py
@@ -219,7 +219,6 @@ class TableTestCase(BaseTestCase):
         test_table = self.session.time_table(period=10000000).update(formulas=["Col1 = i", "Col2 = i * 2"])
         result_table = test_table.snapshot()
         self.assertEqual(test_table.schema, result_table.schema)
-        self.assertGreaterEqual(test_table.size, result_table.size)
 
     def test_snapshot_when(self):
         source_table = (self.session.time_table(period=10_000_000)

